module/PowerNSX.psm1

```diff
@@ -27667,6 +27667,33 @@ function Set-NsxLoadBalancerPoolMember {
 
     This cmdlet configures the state of the specified LoadBalancer Pool Member.
 
+    .EXAMPLE
+    Get-NsxEdge testedge | Get-NsxLoadBalancer | Get-NsxLoadBalancerPool pool1 |
```

Comment:
May be write a note somewhere with the list of variable can be use for Edge, LoadBalancer... (for a better doc)